docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,259 @@
+    `Sensitivity <https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`__ / TPR - how many samples were classified
+    correctly as positive out of the total number of positive samples. Can be also viewed as the fraction of the
+    relevant samples that were spotted by the model.
+
+
+.. image:: /_static/images/general/PrecisionRecall_wikipedia.svg.png
+   :alt: Precision and recall
+   :align: center
+
+Precision and Recall by Walber - Own work, CC BY-SA 4.0,
+`source <https://commons.wikimedia.org/w/index.php?curid=36926283>`__
+
+
+*   `Specificity <https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`__/ TNR - how many samples were classified
+    correctly as negative out of the total number of negative samples. Complementary to sensitivity.
+
+.. image:: /_static/images/general/Sensitivity_and_specificity_wikipedia.svg.png
+   :alt: Specificity and sensitivity
+   :align: center
+
+Specificity and Sensitivity by FeanDoe - Modified version from Walber&#039;s Precision and Recall
+`source <https://commons.wikimedia.org/w/index.php?curid=94134880>`__
+
+*   `F-1 <https://en.wikipedia.org/wiki/F-score>`__ - combines the precision and recall into one metric by taking their
+    harmonic mean. More robust to class imbalance than accuracy.
+*   `AUC <https://developers.google.com/machine-learning/crash-course/classification/roc-and-auc>`__  - The area under
+    the curve of the ROC graph. For the previous classification metrics, a threshold on the model output was chosen
+    above which the sample is classified as positive and below as negative. The AUC takes into account all possible
+    thresholds.
+
+Object Detection
+----------------
+*   `Jaccard Index <https://en.wikipedia.org/wiki/Jaccard_index>`__ / IoU - the ratio between the overlapping area of
+    predicted and the actual bounding box and the union of their areas. The most intuitive metric for object detection.
+*   mAP - mean average precision. The mean of the average precision per class over the classes. Calculated at a certain
+    threshold, usually 0.5. Commonly used for benchmarking object detection models. A good explanation of how it is
+    calculated can be found
+    `here <https://towardsdatascience.com/breaking-down-mean-average-precision-map-ae462f623a52>`__.
+*   mAR - mean average recall. The mean of the average precision per class over the classes. Summed over the threshold
+    range [0.5, 1].
+
+How can I use metrics with Deepchecks?
+======================================
+The metrics used in deepchecks are in accordance with sklearn scorer API for tabular checks and with Torch Ignite metric
+API for vision checks.
+
+Default metrics
+---------------
+All of our relevant checks come with default metrics.
+
+The default scorers by task type are:
+
+Tabular:
+
+Classification, binary or multiclass averaged over the classes:
+
+*   Accuracy
+*   Precision
+*   Recall
+
+Multiclass classification per class:
```

Comment:
Describe the different averaging options for those metrics (macro,micro etc..)
Also mention their string name such as "f1_micro"

Reply:
@Nadav-Barak I think it's too much information, we want to keep it short and simple

Reply:
I don't agree about keeping it short and simple... the main target of this guide is so the user will be able to understand how to use the alternative_metrics

Reply:
I think it is important, if a user pass f1 score to a multilabel task our suite will fail as he must specify an averaging option